src/CarbonAware.CLI/CarbonAwareCLI.cs

```diff
@@ -13,10 +15,11 @@ class CarbonAwareCLI
     public static async Task<int> Main(string[] args)
     {
         ServiceProvider serviceProvider = BootstrapServices();
+        ResourceManager resourceManager = new ResourceManager("CarbonAware.CLI.CommandOptions", Assembly.GetExecutingAssembly());
 
         var rootCommand = new RootCommand()
         {
-            Description = "CLI for retrieving data using Carbonaware SDK"
+            Description = resourceManager.GetString("rootCommandDesc")
```

Comment:
`rootCommandDescription` like the other tags and descriptors in the string library